documentation/src/main/doc/modules/ROOT/pages/getting-started.adoc

```diff
@@ -79,15 +79,18 @@ Running the previous example should give the following output:
 
 [NOTE]
 ====
-SmallRye Reactive Messaging uses SLF4J as logger.
-Check the https://www.slf4j.org/manual.html[SLF4J documentation] to configure the logs.
+SmallRye Reactive Messaging uses JBoss-logger as logger.
+
 For testing purpose, you can simply add the following dependency to your `pom.xml` file:
 [source,xml]
 ----
 <dependency>
-  <groupId>org.slf4j</groupId>
-  <artifactId>slf4j-simple</artifactId>
-  <version>1.7.30</version>
+    <groupId>org.jboss.slf4j</groupId>
+    <artifactId>slf4j-jboss-logging</artifactId>
+    <version>${slf4j-jboss-logging.version}</version>
+    <scope>test</scope>
 </dependency>
 ----
-====
+
+Check the https://www.slf4j.org/manual.html[SLF4J documentation] to configure the logs.
```

Comment:
I'm confused now... Do you mean that I need to configure slf4j, but I don't have any "backend" defined?

Reply:
Yea like I said I am not sure how this will work, but will update this in the next PR once I go through all examples.

Reply:
Ok, I removed all reference to slf4j and showed how to configure logging with JUL. I also fixed #519